tests.py

```diff
@@ -170,7 +170,7 @@ def test_advanced_add(self):
         with open(self.config_file) as f:
             # check that property is really flushed out to the config?
             content = f.read().encode('ascii')
-            self.assertIn(b'identityfile "/tmp/idfilecheck.rsa"', content)
+            self.assertIn(b'IdentityFile "/tmp/idfilecheck.rsa"', content)
             self.assertIn(b"stricthostkeychecking yes", content)
```

Comment:
Yeah, I wonder why dump() isn't respecting the camelcase of the --o option. Need to dig into it further, even though it looks like you've got the lower_to_original function implemented there.